tests_integration/cli/write/invalid.js

```diff
@@ -0,0 +1 @@
+this is invalid!
```

Comment:
This file needs to be ignored from eslint

Reply:
Oops, forgot to turn off this one.

Reply:
We should be able to add `/tests_integration/cli/` to .eslintignore and get rid of all `eslint-disable` comments in `tests_integration/cli/`.

Reply:
@ikatyang Are you going to remove those `eslint-disable` comments, or do you want me to? :)

Reply:
I'm removing them. 😎